analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ParameterNameMatchesOriginal.vb

```diff
@@ -60,3 +60,58 @@ Public Class IgnoreCaseChange
     End Sub
 
 End Class
+
```

Comment:
Seems to me there are no cases for VB where the parameter name is only different by the case. for example: somePARAMETER and SOMEparameter.